public/usage-examples/graphics/draw_triangle/Simple_Red_Triangle.py

```diff
@@ -0,0 +1,15 @@
+from splashkit import *
+
+def main():
+    window = create_window("Simple Red Triangle", 800, 600)
+
+    while not window.close_requested:
+        process_events()
+        clear_screen(color_white())
+        fill_triangle(color_red(), 400, 100, 600, 500, 200, 500)
+        refresh_window(window)
+
+    close_window(window)
+
+if __name__ == "__main__":
+    main()
```

Comment:
Similar here, there is no need to do `def main()` and the `if __name__` it can be done without it. Have a look at how other python files are on splashkit usage examples.

In addition remove the while not close requested, this isn't in the C++ version or C# top level.